services/121-service/test/metrics/__snapshots__/get-program-stats.test.ts.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Get program stats should successfully get program stats 1`] = `
```

Comment:
In this case I prefer testing with an objectContaining and sort of check the attributes dynamically. E.g calculated the cashDisbursed by registration times number of payments etc. 

I feel like snapshots are a bit less readable and bit more brittle. For example if we change the programIds of of our NLRC-seed this snapshot will break.